packages/chatbot-server-mongodb-public/environments/production.yml

```diff
@@ -81,3 +81,15 @@ probes:
 defaultAlerts:
   enabled: true
   email: "chatbot-alerts@mongodb.com"
+
+prometheusRules:
```

Comment:
can you also add this to staging server? 

and also, would be good to test out on staging somehow...maybe temporarily break by changing an env var, then run a non-small load against the server to make sure it works. perhaps could do in the `performance-test` directory, which uses a load testing framework to make this easy

Reply:
> can you also add this to staging server?

Yup - added in this PR just now

> would be good to test out on staging somehow

I think to get the staging environment to actually use it I'll need to push the updated staging config to `main`. Then I can point a test suite at staging and verify. Just noting the plan here + seeing if you have any alternative ideas.